data/src/main/java/com/mashup/data/source/remote/dto/responsebody/LoginResponseBody.kt

```diff
@@ -0,0 +1,8 @@
+package com.mashup.data.source.remote.dto.responsebody
+
+data class LoginResponseBody(
```

Comment:
요기랑 RequestBody Json 파서 안붙여도되나? moshi!

Reply:
붙어야되는걸로 알고있어 나두

Reply:
그리구 이건 같이 확인해봐야 할 거같은데, 나도 Moshi를 처음 써봐서 자료를 좀 찾아보니까
```suggestion
@JsonClass(generateAdapter=true)
data class LoginResponseBody()
```
요런식으로 Annotation 추가를 해주어야 직렬화랑 역직렬화가 되는 것 같거든. 
그렇다면 이 Annotation은 Response에도 추가가 되어야할 것 같아!! 우리 Response에는 이게 안붙어있는 것 같더라구

추가로, `@field:Json(name="~")`랑 `@Json(name="~")`의 차이도 궁금합니당
이 PR에서 다룰 내용은 아니지만 수정이 필요하다하면 하는 김에 Response도 같이 수정해도 될 것 같아서!!

ref. https://kotlinworld.com/117

Reply:
2788e5fc38b646db9ce66560777031dd2712a3f3 3f1b0f0d9341e4279928e1b6b3aa3dc74781f572
generateAdapter를 하려면 build.gradle에 moshi codegen을 추가해야 하더라고! 추가하고 generateAdapter 세팅 완 .
https://github.com/square/moshi#codegen

@field:Json(name="~")랑 @Json(name="~")의 차이는..
https://stackoverflow.com/questions/56976656/moshi-in-kotlin-json-vs-fieldjson 이거를 읽어보긴 했는데 사실 잘 와닿진 않더라고..?

Reply:
> Serialization does not happen properly when I use @Json in the fields but it started working after changing to @field:Json.

일단 @Json만 있으면 직렬화가 잘 안되었다는데 바꿔보는건 어떠신지~

Reply:
접수~ ca167e58edc62eb5ddd961edd5f85ecbc1f670e3